### ISSUE TITLE
Support another package sources (Private Nuget )

### ISSUE THREAD

**Issue by amir135** [rsuter/NuGetReferenceSwitcher#54]
We use own Nuget package sources and can not use this tool for switch between private Nuget plugin and project . 
Can we use this tool with another package sources?

**Comment by BigHoss** [comment]
Yes you can, you just need a `nuget.config` file in the root of your project [Documentation of nuget and config files](https://docs.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior#creating-a-new-config-file)

>Edit: I am not quite sure if it will really work with the reference switcher, I thought this was the [dnt repo](https://github.com/RicoSuter/DNT) (where it definitely works)
